helm/alfresco-content-services/tests/deployment-repository_test.yaml

```diff
@@ -17,6 +17,18 @@ tests:
           value: RELEASE-NAME-alfresco-cs-repository
         template: deployment-repository.yaml
 
+  - it: Log4j & Log4j2 test
+    values: *testvalues
+    set:
+      repository.extraLogStatements.testkey: testvalue
+    asserts:
+      - contains:
+          path: spec.template.spec.containers[0].envFrom
+          content:
+            configMapRef:
+              name: RELEASE-NAME-alfresco-cs-repository-configmap
+        template: deployment-repository.yaml
```

Comment:
You are testing the envFrom section contains a reference to the `RELEASE-NAME-alfresco-cs-repository-configmap` cm... That doesn't make sense to me.
You should be testing: 
* the volumemount exists
* the cm has the right entries  with the right values.